### ISSUE TITLE
Node compilation error under new virtual environment

### ISSUE THREAD

**Issue by noiseunion** [actions/virtual-environments#4647]
### Description

We have a GH Action that runs our ES Lint process.  It was working under `Version: 20211122.1` but is now blowing up under `Version: 20211129.1`.

From my understanding of this previous issue: https://github.com/actions/virtual-environments/issues/2505

There is no way for me to pin back to the previous virtual environment.

We are now seeing this on every run of the ES-Linter action:

```
Building fresh packages...
error /home/runner/work/titan/titan/node_modules/node-sass: Command failed.
Exit code: 1
```

This is also followed by a long list of `gyp` lines ultimately culminating in the following:

```
make: Leaving directory '/home/runner/work/titan/titan/node_modules/node-sass/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/runner/work/titan/titan/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (node:events:390:28)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12)
gyp ERR! System Linux 5.11.0-1021-azure
gyp ERR! command "/usr/local/bin/node" "/home/runner/work/titan/titan/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--verbose" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
gyp ERR! cwd /home/runner/work/titan/titan/node_modules/node-sass
gyp ERR! node -v v16.13.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 
Build failed with error code: 1
Error: Process completed with exit code 1.
```

This appears to be a problem with the new virtual environment to me, but I can't verify since I can't pin to the older version and see if the world is happier. 

Any help would be appreciated.

### Virtual environments affected

- [ ] Ubuntu 18.04
- [X] Ubuntu 20.04
- [ ] macOS 10.15
- [ ] macOS 11
- [ ] Windows Server 2016
- [ ] Windows Server 2019
- [ ] Windows Server 2022

### Image version and build link

Version: 20211129.1

### Is it regression?

20211122.1

### Expected behavior

I expected the action to not be blowing up while trying to install node dependencies

### Actual behavior

The action blows up and fails due to errors in the process of setting up the environment to run our linters

### Repro steps

not sure I know how to tell you this without sending maybe the package.json file over and a copy of the GH action?

**Comment by kis-xt** [comment]
What I see that previous build used [Node 14.18.1 ](https://github.com/actions/virtual-environments/blob/ubuntu20/20211122.1/images/linux/Ubuntu2004-README.md) but this minor image update change node version to [Node 16.13.0](https://github.com/actions/virtual-environments/blob/ubuntu20/20211129.1/images/linux/Ubuntu2004-README.md)

**Comment by catthehacker** [comment]
https://github.com/actions/virtual-environments/issues/4446

**Comment by noiseunion** [comment]
@kis-xt @catthehacker - thank you for the quick responses.  Should this not have been a bit more prominently called out here: [Changelog blog](https://github.blog/changelog/)?

> High Impact Changes (ex. breaking changes, new or deprecated environments) will be posted to the GitHub Changelog on our blog and on twitter.

I went there to look for details before submitting this and I did not see this change mentioned anywhere. Not saying that I didn't miss it....but it doesn't seem to be easy to see in the location we are told to go look for such a change.
